spk/domoticz/src/dsm-control.sh

```diff
@@ -16,6 +16,14 @@ PORT="8084"
 
 start_daemon ()
 {
+    insmod /lib/modules/usbserial.ko
+    insmod /lib/modules/ftdi_sio.ko
+    insmod ${INSTALL_DIR}/modules/cp210x.ko
```

Comment:
You might be right. My concern was mainly if this would have the potential of leaving stuff (specifically, a loaded cp210x.ko module) behind after removing the package. I really don't know the answer to that.